src/states/node.rs

```diff
@@ -918,6 +922,40 @@ impl Node {
         }
     }
 
+    // Tell all neighbouring sections that our member list changed.
+    // Currently we only send this when nodes join and it's only used to add missing members.
+    fn send_section_update(&mut self) {
+        trace!("{:?} Sending section update", self);
+        let names = self.peer_mgr.routing_table().our_names();
+        // TODO: why does the method return a HashSet? A Vec is fine for us
```

Comment:
I think elsewhere it uses a `HashSet`. Probably it comes down to spelling out the complicated iterator return type again.